src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user: { name, email } }) => {
+  return (
```

Comment:
It will be more readable
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => {
  const { name, email } = user;
  
  return (
```